src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -91,6 +92,10 @@ public Command parseCommand(String userInput) throws ParseException {
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
 
+        case IngredientResetAllCommand
+                .COMMAND_WORD:
```

Comment:
I think that they should be on the same line.

Reply:
it automatically become the next line when I save.. so can I leave it as that?

Reply:
I think it is better to be placed on the same line to standardise with the other case statements.

Reply:
0k thanks